### ISSUE TITLE
CLI Client - Implement functionality to register, log in and log off (API v1.1.0)

### ISSUE THREAD

**Issue by kerrambit** [kerrambit/RSSFeedify#26]
**CLI Client - Implement functionality to register, login and log off (API v1.1.0)**

This must be done before dev-authN/authZ can be merged into main so the client is up to date with API.
In the future, the client must be moved to the independent repository or will be deprecated - the goal is to make a new repo for the GUI application which will work with API - client CLI code can provide at least some basic functionality (some base classes etc).

**Comment by kerrambit** [comment]
Besides main implementation of login, register, and logoff, here are some other minor issues that are better to be tracked independently:

- [ ] https://github.com/kerrambit/RSSFeedify/issues/31
- [ ] https://github.com/kerrambit/RSSFeedify/issues/32
- [ ] and organize classes and interfaces in RSFeedifyClientCore into folders and namespace

**Comment by kerrambit** [comment]
Classes organization will be done in another issue.
